### PR TITLE
Add new endpoints to get info about my investments

### LINKED ISSUE #44: Add new api for my investments

### PR DESCRIPTION
**Pull request by lukasnovy**
Mark old my investments endpoints as deprecated. Current state of development
Issue #44

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1037,6 +1104,135 @@ Available types are:
...
+            - Members
+                - `EMPLOYMENT`
+                - `SELF_EMPLOYMENT`
+                - `MATERNITY_LEAVE`
+                - `ENTREPRENEUR`
+                - `STUDENT`
+                - `PENSION`
+                - `LIBERAL_PROFESSION`
+                - `UNEMPLOYED`
+                - `OTHER`
```
Comment (triceo) [inline apiary.apib:1169]: FYI I have previously seen `OTHERS_MAIN` here as well.

File: apiary.apib [hunk 2]
```diff
@@ -1037,6 +1104,135 @@ Available types are:
...
+        - paidInstalments: `2` (number, required) - Number of already paid instalments
+        - longestDaysDue: `64` (number, required) - Maximum number of days after a due date of the oldest unpaid instalment in whole history of loan
+        - daysSinceLastInDue: `128` (number, required) - Number of days since last due instalment has been repaid
+        - instalmentsCurrentlyInDue: `1` (number, required) - Number of instalments that are currently in due
+        - loanHealthInfo (enum, required)
+            - Members
+                - `CURRENTLY_IN_DUE` - Last payment is not paid on time
+                - `HISTORICALLY_IN_DUE` - There is delayed payment in past but currently borrower is paying on time
+                - `HEALTHY` - There is no delayed payment during whole time
+- smpSellInfo (object, required)
```
Comment (triceo) [inline apiary.apib:1229]: I have brought this up in the mailing list earlier, but let me reiterate: having `smpSellInfo` only available on a separate endpoint forces third-party apps to hit Zonky API too often, causing `HTTP 429 Too Many Requests`. The same goes for `healthStats`.

This PR suggests that the API is still being evolved. Perhaps this is something that could be fixed?
If the API intends to limit how many API requests the users can make, then it makes sense for the API to be designed to limit the amount of requests necessary.

Comment (lukasnovy) [reply apiary.apib:1229]: I'm aware of your concerns and also the email discussion. This PR describes current situation and endpoints, that can be used right now. It was designed primary for our usage and we are doing more improvements for public usage. Unfortunately these additional features are blocked by some other tasks we need to finish first and takes longer that we expected. I hope that we will be able to move requested fields from detail to list endpoint before we remove the old endpoints

File: apiary.apib (hunk 1 above)
Comment (lukasnovy) [reply apiary.apib:1169]: I will check it again, but I believe that `OTHERS_MAIN` is not a valid option and was returned by some loans as a result of a bug

**Pull request merged**